storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3439,6 +3438,12 @@ static int rocksdb_init_func(void *const p) {
 
   DBUG_ASSERT(!mysqld_embedded);
 
+  if (rocksdb_db_options->max_open_files > (long) open_files_limit) {
+    sql_print_information("RocksDB: rocksdb_max_open_files should not be "
```

Comment:
Does it need to say that rocksdb_max_open_files effective value is being adjusted to open_files_limit ?

Reply:
Yeah, that would be more clear. IIRC I copied some of this from InnoDB and probably altered it in trying to fit it into here.